main.go

```diff
@@ -152,12 +150,11 @@ func main() {
 		router.Handle("/cookies{uri:.*}", createReverseProxy("cookies", cookiesControllerURL))
 	}
 
-	if cfg.DatasetRoutesEnabled {
-		router.Handle("/datasets/{uri:.*}", createReverseProxy("datasets", datasetControllerURL))
-		router.Handle("/feedback{uri:.*}", createReverseProxy("feedback", datasetControllerURL))
-		router.Handle("/filters/{uri:.*}", createReverseProxy("filters", filterDatasetControllerURL))
-		router.Handle("/filter-outputs/{uri:.*}", createReverseProxy("filter-output", filterDatasetControllerURL))
-	}
+	router.Handle("/datasets/{uri:.*}", createReverseProxy("datasets", datasetControllerURL))
+	router.Handle("/feedback{uri:.*}", createReverseProxy("feedback", feedbackControllerURL))
```

Comment:
Either need to make sure that the feedbackControllerURL is set to the value of the datasetControllerURL in the secrets everywhere, or surround this route in a feature flag again to only use the new value if enabled. 

Right now, if merged into develop, this will break.

Reply:
Or even just change this back to datasetControllerURL for now if the feedback controller stuff is still outstanding in branches